### DISCUSSION TITLE
Cracked Servers UUID Spoofing

### DISCUSSION THREAD

**Discussion by TechPro424** [axieum/authme#27] [Q&A]
Can this mod prevent uuid spoofing when logging in to cracked servers?

**Comment by axieum** [comment]
Hi, here is where the UUID gets computed and set for an offline account:

https://github.com/axieum/authme/blob/e8e538afb72de6d3c50fe8d3fafb97807a69fb9c/src/main/java/me/axieum/mcmod/authme/util/SessionUtil.java#L146-L148

Did you want to be able to set this UUID value yourself?

By playing on an offline-mode server, you opt-out of all player verifications. There is no way to prevent others from logging in as yourself by playing on an offline-mode server.

**Comment by BytelyPlay** [comment]
If I understand you correctly, this modification has nothing to do with what you're talking about.
You can't choose your UUID in Offline-Mode servers to start with. You can choose your username, and the server computes your UUID the same way Axieum did here.
This is a client-sided modification, it can't do any verification or authentication on the server.

On a side-note, I wanted to say, that I really don't like piracy, and REALLY don't support piracy, but this is one of the times I'll just answer.
Piracy is also illegal in most countries. Pirate Minecraft at your own risk.
It carries fines in most countries.
